### ISSUE TITLE
Search Window Problem for "Added Search Bar in Toolbar Option"

### ISSUE THREAD

**Issue by s3dbw** [MrAlex94/Waterfox#3038]
### What happened?

When using the separate search window, once you start to type something, the search box will not let you place the cursor anywhere else within the search bar window, so for example if you type "Sarrch" instead of "search" you cant place the cursor back into the word and make corrections, you need to completely delete and try again. This is only appears on Waterfox (GS5.1.6 64bit). I have tried this on Firefox and no problems, you can edit anywhere within the search box?

### Reproducible?

- [X] I have tested this on Firefox.

### Version

G5

### What platform are you seeing the problem on?

Windows

**Comment by MarjaE2** [comment]
Also occurs in MacOS. If you have trouble typing and need to correct what you've typed, this makes search much harder.

**Comment by MarjaE2** [comment]
May be a duplicate of bug 2981.

**Comment by irontechmonkey** [comment]
> May be a duplicate of bug 2981.

Agreed, it does seem to be the same issue (here is a link to 2981: https://github.com/WaterfoxCo/Waterfox/issues/2981). That ticket and this one have both been flagged as "bugs" and this one (the one we are now in) has more notes so maybe this should be the active ticket. 

In any event, Reproduced in Windows 10 Waterfox G5.1.10 (64-bit); does not occur in Firefox 116.0.2 (64-bit). 

Mote details:

- Occurs in search box (when it is placed in in toolbar, menu bar, tab bar, status bar, anywhere); the box does not respond to mouse input except to select the contents if focus has been placed elsewhere then returned.

- While text is being typed and while the search text box remains in focus the mouse does nothing in the box, not even select text. Keyboard edit commands work fine.

- If aother filed is clicked with mouse or if Tab is used to navigate to next field (thereby removing focus from the search box) then if the search box is click on with mouse the contents will be selected just as if Tab was used to navigate to it.

- Upon return of focus to the search box, the mouse will still do nothing but keyboard cursor keys can move through the text in the box and keyboard commands work fine as they could in previous step.
